test/energy_scan/mult_and_spectra.py

```diff
@@ -306,7 +306,7 @@ def merge_basic_spectra(files_to_read_list, files_to_write):
         files_to_read_list = zip(*(iter(args.input_files),) * 8)
         BulkObservables.merge_basic_spectra(files_to_read_list, args.output_files)
     else:
-        pdg_list = [211,-211,111,321,-321,2212,-2212,3122,-3122,3312,-3312,3334,-3334]
+        pdg_list = [211,-211,111,321,-321,2212,-2212,3122,-3122,1000010020,-1000010020,3312,-3312,3334,-3334,3212,-3212]
```

Comment:
Did you add the Deuterons on purpose? I do not think they should be part of the default energy scan.

Reply:
I would actually be happy, if deuterons are added to the energy scan. SMASH turns out to reproduce deuteron dN/dy very nicely. Experimental data is all in the repository, the only thing to do is to change SMASH cross section cutoff to 2000 mb.

But for now you can remove the deuteron.

Reply:
Maybe let's compute deuteron quantities, but not plot them. Will that be a satisfactory solution?

Reply:
> Maybe let's compute deuteron quantities, but not plot them. Will that be a satisfactory solution?

Yes, I can live with that. But I would not change the default SMASH x-section cutoff because it increases the runtime tremendously. Leaving the computation of the quantities is however fine for me.